Resources/Textures/Clothing/Uniforms/Jumpsuit/ert_engineer.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation then edited at commit https://github.com/tgstation/tgstation/commit/c838ba21dae97db345e0113f99596decd1d66039",
```

Comment:
The test is failing because of a error in master so not your fault, still, it would be better if you link the dmi **in** the commit instead of just the commit (for more details see https://github.com/space-wizards/space-station-14/issues/13381)

Reply:
Alright, I understand your point, however it's bit of a rough place for me as I'm simply editing another file of ours `hos_alt.rsi` that uses the same copyright path, thus I do not actually know the origin of the sprite.

Reply:
Wait then where did you get the new sprites from?

Reply:
I took the sprites from `hos_alt.rsi` the suit already in our files, then hand edited it to produce these.

Reply:
Found it using https://scrubby.melonmesa.com/ . You should put something like "Edited from https://github.com/tgstation/tgstation/blob/c838ba21dae97db345e0113f99596decd1d66039/icons/mob/clothing/under/security.dmi by Nairodian (Github) for SS14"

Reply:
Alright, should be better now. I was a little bit vague but the main idea is there.